DragaliaAPI/Services/HelperService.cs

```diff
@@ -0,0 +1,2344 @@
+                            ability_3_level = 0,
+                            ex_ability_level = 1,
+                            ex_ability_2_level = 1,
+                            skill_1_level = 1,
+                            skill_2_level = 0,
+                            is_unlock_edit_skill = true
+                        },
+                        support_dragon = new() { dragon_key_id = 0, },
+                        support_weapon_body = new() { weapon_body_id = 0, },
+                        support_talisman = new() { talisman_key_id = 0, },
+                        support_crest_slot_type_1_list = new List<AtgenSupportCrestSlotType1List>()
+                        {
+                            new() { ability_crest_id = 0 },
+                            new() { ability_crest_id = 0 },
+                            new() { ability_crest_id = 0 },
+                        },
+                        support_crest_slot_type_2_list = new List<AtgenSupportCrestSlotType1List>()
+                        {
+                            new() { ability_crest_id = 0 },
+                            new() { ability_crest_id = 0 },
+                        },
+                        support_crest_slot_type_3_list = new List<AtgenSupportCrestSlotType1List>()
+                        {
+                            new() { ability_crest_id = 0 },
+                            new() { ability_crest_id = 0 },
+                        },
+                        guild = new() { guild_id = 0, guild_name = "Guild" }
+                    },
+                    new()
+                    {
+                        viewer_id = 1001,
+                        name = "Nightmerp",
+                        level = 250,
+                        last_login_date = DateTimeOffset.UtcNow - TimeSpan.FromDays(1),
+                        emblem_id = 10250305,
+                        max_party_power = 9999,
+                        support_chara = new()
+                        {
+                            chara_id = Charas.GalaEmile,
+                            level = 80,
+                            additional_max_level = 0,
+                            rarity = 5,
+                            hp = 789,
+                            attack = 486,
+                            hp_plus_count = 100,
+                            attack_plus_count = 100,
+                            ability_1_level = 2,
+                            ability_2_level = 2,
+                            ability_3_level = 2,
+                            ex_ability_level = 5,
+                            ex_ability_2_level = 5,
+                            skill_1_level = 3,
+                            skill_2_level = 2,
+                            is_unlock_edit_skill = true
+                        },
+                        support_dragon = new()
+                        {
+                            dragon_key_id = 0,
+                            dragon_id = (int)Dragons.GalaBahamut,
```

Comment:
could you change the properties like this one in the supportchara type to be Dragons/AbilityCrests/etc to avoid this cast